cmd/statusd/main.go

```diff
@@ -131,10 +178,11 @@ func printUsage() {
 	fmt.Fprintln(os.Stderr, "Usage: statusd [options]")
 	fmt.Fprintf(os.Stderr, `
 Examples:
-  statusd               # run status node with defaults
-  statusd -networkid 4  # run node on Rinkeby network
-  statusd -datadir /dir # specify different dir for data
-  statusd -ipc          # enable IPC for usage with "geth attach"
+  statusd                      # run status node with defaults
+  statusd -networkid 4         # run node on Rinkeby network
+  statusd -datadir /dir        # specify different dir for data
+  statusd -ipc                 # enable IPC for usage with "geth attach"
+  statusd -cli localhost:12345 # enable connection by local statusd-cli on port 12345
```

Comment:
`cli` is defined as `bool` in `cmd/statusd-cli/main.go`.

Reply:
Yeah, fixed, old example from first implementation. Thanks.